status.go

```diff
@@ -4,11 +4,12 @@ import "time"
 
 // StatusResult describes the current state of a transaction.
 type StatusResult struct {
-	Languages            []string            `json:"language_list"`
-	MinutesLeftOnReserve float64             `json:"minutes_left_on_reserve"`
-	Trolley              Trolley             `json:"trolley_contents"`
-	ReserveDatetime      time.Time           `json:"reserve_iso8601_date_and_time"`
-	PurchaseDatetime     time.Time           `json:"purchase_iso8601_date_and_time"`
-	CurrencyDetails      map[string]Currency `json:"currency_details"`
-	Status               string              `json:"transaction_status"`
+    Languages            []string            `json:"language_list"`
+    MinutesLeftOnReserve float64             `json:"minutes_left_on_reserve"`
+    Trolley              Trolley             `json:"trolley_contents"`
+    ReserveDatetime      time.Time           `json:"reserve_iso8601_date_and_time"`
+    PurchaseDatetime     time.Time           `json:"purchase_iso8601_date_and_time"`
+    CurrencyDetails      map[string]Currency `json:"currency_details"`
+    Customer             Customer            `json:"customer"`
```

Comment:
n.b. The actual change - Jakub likes to bury things in linters

Reply:
It's not me - it's because the code is not well formatted ;) But don't worry, I'll fix it ;)